### PR TITLE
Need to use utf-8

### PR DESCRIPTION
**Pull request by lyriccoder**
https://github.com/yegor256/aibolit/issues/40

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#41](https://github.com/yegor256/aibolit/issues/41) is assigned to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda), here is [why](https://www.0crat.com/footprint/CT5UB7XAP/9e2264f9-169f-4163-8a97-c5ccd5011984); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @yegor256[/z](https://www.0crat.com/u/yegor256) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: scripts/04-find-patterns.py [hunk 1]
```diff
@@ -1,92 +0,0 @@
-# The MIT License (MIT)
```
Comment (carlosmiranda) [inline scripts/04-find-patterns.py]: @lyriccoder why is this file deleted?

Comment (lyriccoder) [reply scripts/04-find-patterns.py]: This file has is not from this merge. I've committed it accidentally, the main fix is just adding an encoding

**Comment by 0crat** [comment]
Job audit: `ARC` didn't merge or close PR, performer @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)[/z](https://www.0crat.com/u/carlosmiranda) didn't make CR comments

**Comment by 0crat** [comment]
Job was finished in 17 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Pull request [#41](https://github.com/yegor256/aibolit/issues/41) was not merged, no payment for `ARC`, see [§28](http://www.zerocracy.com/policy.html#28)

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @carlosmiranda[/z](https://www.0crat.com/u/carlosmiranda)
